docs/apps_tutorial.md

```diff
@@ -887,35 +909,40 @@ docker push oxedions/app-interactive_shell:v1
 
 Once image is pushed, add application into Jarvice as before. Note that this time, we didn't base64 encoded a png image, so default image is used.
 
-When clicking on application card, you should now have:
+When you click on the application card, you should now view the following:
 
-![app_interactive_shell_step_1](img/apps_tutorial/app_interactive_shell_step_1.png)
+<!--![app_interactive_shell_step_1](img/apps_tutorial/app_interactive_shell_step_1.png)-->
+![app_interactive_shell_step_1](img\apps_tutorial\New_Images\Img21_GottyShell.png){: style="border: 0.5px solid black; width: 450px;"}
```

Comment:
Same here, check \ vs /

Reply:
There are other places too, just look for `img\` pattern to get them all

Reply:
Thank you for the feedback @oxedions ! Incorporated it! All the \ in the image paths are changed to /.